documents/Specification/MaterialX.Specification.md

```diff
@@ -708,6 +708,14 @@ The type of the &lt;image> node determines the number of channels output, which
     * `realworldtilesize` (vector2): the real-world size of a single square 0-1 UV tile, with unittype "distance".  A `unit` attribute may be provided to indicate the units that `realworldtilesize` is expressed in.
     * `filtertype` (uniform string): the type of texture filtering to use; standard values include "closest" (nearest-neighbor single-sample), "linear", and "cubic".  If not specified, an application may use its own default texture filtering method.
 
+<a id="node-latlongimage"> </a>
+
+* **`latlongimage`** (NG): samples an equiangular map along a view direction with adjustable latitudinal offset.
+    * `file` (uniform filename): the URI of an image file.  The filename can include one or more substitutions to change the file name (including frame number) that is accessed, as described in the [Filename Substitutions](#filename-substitutions) section above.
+    * `default` (float or color<em>N</em> or vector<em>N</em>): a default value to use if the `file` reference can not be resolved (e.g. if a &lt;_geometry token_>, [_interface token_] or {_hostattr_} is included in the filename but no substitution value or default is defined, or if the resolved `file` URI cannot be read), or if the specified `layer` does not exist in the file.  The `default` value must be the same type as the `<image>` element itself.  If `default` is not defined, the default color value will be 0.0 in all channels.
+    * `viewdir` (vector3): the view direction determining the value sampled from the projected equiangular map.
+    * `rotation` (float): the latitudinal sampling offset.
```

Comment:
Should this language read "longitudinal sampling offset", since the rotation amount is added to the longitude in the graph definition?

Reply:
Good catch! Fixed the typo.